widgets/DcaWizard.php

```diff
@@ -302,9 +302,8 @@ public function getDcaWizardOptions()
      */
     public function getButtonHref()
     {
-        return ampersand(\Environment::get('base')
-            . TL_SCRIPT
-            . '?'
+        return StringUtil::ampersand(Environment::get('base')
```

Comment:
can we generate a real route here? Because the `/contao` prefix can be changed in the configuration

Reply:
Changed in 42b782f.